packages/terra-clinical-result-view/src/ResultView.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconDocuments from 'terra-icon/lib/icon/IconDocuments';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconUnexpected from 'terra-icon/lib/icon/IconUnexpected';
+import styles from './ResultView.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   *  An array of one or more results.
+   */
+  results: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.number,
+    unit: PropTypes.string,
+    normalcy: PropTypes.node,
```

Comment:
Can you do typeOf Icon rather than node?

Reply:
enforcing an svg icon